packages/storybook-html/src/components/hero/hero.docs.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
+
+<Meta title="Components/Hero/All" />
+
+# Hero
+<StatusBadge.Development />
+
+HERO 
+
+<Canvas withSource="none">
+  <Story id="components-hero" height={505} />
+</Canvas>
+
+### Hero default
+
+```html
+<div class="hero">  
+  <div class="hero__image">
+    <div class="hero__image--small">
+      <img src="https://picsum.photos/375/250" />
+    </div>
+    <div class="hero__image--large">
+      <img src="https://picsum.photos/1920/540" />
+    </div>
+  </div>
+  <div class="hero__content uq-grid">
+    <div class="hero__text uq-grid__col uq-grid__col--6">
```

Comment:
On smaller viewports, the text could go full width but at the moment its retaining the 6 col width